apps/workflows/src/cron/tinybird.ts

```diff
@@ -0,0 +1,119 @@
+import { db, eq } from "@openstatus/db";
+import { type WorkspacePlan, workspace } from "@openstatus/db/src/schema";
+import { env } from "../env";
+
+import readline from "node:readline";
+
+// Function to prompt user for confirmation
+const askConfirmation = async (question: string): Promise<boolean> => {
```

Comment:
This is to avoid doing any accidental deletion.

![CleanShot 2024-11-19 at 19 52 45@2x](https://github.com/user-attachments/assets/1a509caa-e5e9-4a20-9117-55555667613c)